applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/pm_static.yml

```diff
@@ -28,7 +28,11 @@ mcuboot_primary_app:
 settings_storage:
   address: 0xf0000
   region: flash_primary
-  size: 0x10000
+  size: 0xf000
+factory_data:
```

Comment:
It is worth double-checking if this won't break Thingy:53 Device Manager. Does switching to another app from the store clears the settings partition? Does it mean that by reducing the size we won't clear it fully when switching to another app?

Reply:
I created two configurations, one is the same as it was, and the other contains factory data partition.